payment/credentials.go

```diff
@@ -17,6 +20,72 @@ const (
 	defaultMaxTokensPerIssuer = 4000000 // ~1M BAT
 )
 
+func decodeIssuerID(issuerID string) (string, string, error) {
+	var (
+		merchantID string
+		sku        string
+		skuOffset  uint64
+	)
+	b, err := base64.StdEncoding.DecodeString(issuerID)
+	if err != nil {
+		return "", "", fmt.Errorf("failed to decode issuerID: %w", err)
+	}
+
+	skuOffsetBuf := bytes.NewReader(b[:8])
```

Comment:
Can we set `8` to a constant? Would be difficult to know that an unsigned int 64 is 8 bytes without prior knowledge

Reply:
using url scheme now in https://github.com/brave-intl/bat-go/pull/303/commits/b361586f76bf794e79f2127c66b215713b9896e4